dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -0,0 +1,148 @@
+
+The xref:../admin_guide/high_availability.adoc#[High Availability] section
+describes how to make this highly available using replicated services.
+
+endif::[]
+
+
+[[using-a-router]]
+== Using a Router
+
+This is the most common way to access the cluster. A router is configured
+to accept external reuqests and proxy them based on the configured routes.
+This is limited to HTTP/HTTPS(SNI)/TLS(SNI) which covers web applications.
+
+An administrator can create a wild-card DNS entry, and then set up a router,
+after that the users can self-service the ingress without having to bother
+the admins. The router has controls to allow the admin to specify whether
+the users can self-provision hostnames, or if they must fit a pattern the
+admin defines. The other solutions require the admin to do it, or it requires
+that they delegate a lot of privilege.
+
+A set of routes can be created in the various projects. The overall
+set of routes is available to the set of routers. Each router selects
+from the set of routes.  All routers see all routes unless restricted
+by labels on the router, this is called router
+xref:../architecture/core_concepts/routes.adoc:#router-sharding[sharding].
+
+The xref:../admin_guide/high_availability.adoc#[High Availability] section
+describes how to configure a router for high availbility service using
+multiple replicas.
+
+[[using-the-loadbalancer]]
+== Using a Service of Type LoadBalancer
+
+There are load balancers available on
+xref:../install_config/configuring_aws.adoc#[AWS] and
+xref:../install_config/configuring_gce.adoc#[GEC] clouds as well as
+xref:../admin_guide/tcp_ingress_external_ports.adoc#[bare metal].
+
+The bare-metal
+xref:../admin_guide/tcp_ingress_external_ports.adoc#[LoadBalancer] allocates
+a unique IP from a configured pool. This limits you to a single ingress IP
+(can be a VIP, but still will be a single machine for initial load balancing).
+It simplifies the admin job, but uses one IP per service.
+
+Load balancing is described
+xref:http://kubernetes.io/docs/user-guide/services/#type-loadbalancer[here].
+
+[[using-externalIP]]
+== Using a Service ExternalIP
+
+The user can manually assign
+xref:http://kubernetes.io/docs/user-guide/services/#external-ips[ExternalIPs]
+to the service. The supplied list of IP addresses are used for load balancing
+incoming requests. The service port is opened on all IP addresses in the list.
+Doing this requires elevated permissions to assign, and manual tracking of the
+IP:ports that are in use.
+
+An externally visible IP can be configured in one of several ways:
+
+- manually configure the externalIPs with a known external ip address.
+- configure externalIPs to a xref:../admin_guide/high_availability.adoc#[VIP]
+that is generated from (VRRP)
+- In a cloud (xref:../install_config/configuring_aws.adoc#[AWS] or
+xref:../install_config/configuring_gce.adoc#[GEC]) use type=LoadBalancer
+- In a xref:../admin_guide/tcp_ingress_external_ports.adoc#[bare metal]
```

Comment:
Consider removing this reference since the above section already makes mention of it.